### ISSUE TITLE
Fix header hierarchy when both multiple rooms and dates in a seminar

### ISSUE THREAD

**Issue by floogulinc** [UMM-CSci/senior-seminar#41]
Currently we end up with two sets of `<h3>` elements in a row where, eg:
<blockquote>
<h3>Sunday, 15 April 2018</h3>
<h3>Sci 3610</h3>

[table of stuff]

<h3>Sci 3650</h3>
</blockquote>

but we should have the secondary ones become `<h4>` elements, eg:

<blockquote>
<h3>Sunday, 15 April 2018</h3>
<h4>Sci 3610</h4>

[table of stuff]

<h4>Sci 3650</h4>
</blockquote>

This is important mainly for accessibility.
